src/View/ThemeResourceLoader.php

```diff
@@ -182,9 +183,11 @@ public function getPath($identifier)
      * @return string Absolute path to resolved template file, or null if not resolved.
      * File location will be in the format themes/<theme>/templates/<directories>/<type>/<basename>.ss
      * Note that type (e.g. 'Layout') is not the root level directory under 'templates'.
+     * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it.
      */
     public function findTemplate($template, $themes = null)
```

Comment:
Doesn't make sense for this to be here anymore, since this only knows about ss templates.

There is a replacement method for this in the new engine but I've opted to keep it private, to dissuade people from trying to deal with file paths for templates - we should be thinking and talking about template files as template "candidates" instead, i.e. "render with `Includes/Header`" and let the template engine figure out through theme inheritance which exact template file that maps to.